### PR TITLE
DSD-523: Updating margins in the TextInput component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-523](https://jira.nypl.org/browse/DSD-523)

## This PR does the following:
- Removes the bottom margins from the `TextInput` and its `HelperErrorText` component.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr721-u03b21jztmtb93o2fi805dj4dilml4jn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/616756194dc8a8153f8f3803

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr721-u03b21jztmtb93o2fi805dj4dilml4jn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/616756194dc8a8153f8f3803

File: src/theme/components/textInput.ts [hunk 1]
```diff
@@ -31,9 +31,8 @@ const input = {
 const TextInput = {
   parts: ["helper", "input", "textarea"],
   baseStyle: {
-    marginBottom: "xs",
     helper: {
-      ...helperTextMargin,
```
Comment (bigfishdesign13) [inline src/theme/components/textInput.ts:36]: You can actually keep this.  The element that needs to be adjusted is the actual `HelperErrorText` component.  By default, the `HelperErrorText` component has a bottom margin.  That bottom margin should be removed from that element altogether or the styles should be rewritten to only add a bottom margin when the `HelperErrorText` element is not the `last-child`.

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
The bottom margin on the `HelperErrorText` element needs some adjustments.  I didn't think about it when I put this ticket together, but this same issue also appears on any input component that uses `HelperErrorText`.  When these components are dropped into the `Form` layout component, the bottom margin on `HelperErrorText` throws things out of whack.

**Comment by EdwinGuzman** [comment]
How about we close this and I make the appropriate update in DSD-509 #725 ?

**Comment by bigfishdesign13** [comment]
> How about we close this and I make the appropriate update in DSD-509 #725 ?

There are two margin values that need to be removed.  Yes, one is on the `HelperErrorText` component, but one is part of the `TextInput` styles.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr721-u03b21jztmtb93o2fi805dj4dilml4jn.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/616756194dc8a8153f8f3803

**Review by bigfishdesign13 — APPROVED** [review]
Simple.  Clean.  Looks good.

**Pull request merged**